sympy/functions/elementary/exponential.py

```diff
@@ -459,15 +459,15 @@ def _eval_nseries(self, x, n, logx):
         r = r.replace((-1)**w, expand_complex((-1)**w))
         return r
 
-    def _taylor(self, x, n, nterms):
+    def _taylor(self, x, nterms):
```

Comment:
If this works, we could have `n` everywhere in `_taylor` instead of `nterms`.

Reply:
Ok that would be good too.